### ISSUE TITLE
aria-hidden advice in checklist is incomplete

### ISSUE THREAD

**Issue by EvertVd** [a11yproject/a11yproject.com#84]
It states in the checklist that aria-hidden should be used to hide content from all users (incl. visual browsers). This is not entirely correct. The spec states that the means of hiding content with this attribute is up to the author, and it further says:

> "Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies only if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies."

In other words, using aria-hidden to just hide content from screenreaders but not from visual browsers is allowed. to hide from all UAs and ATs it would be better to use the HTML attribute "hidden"
